### ISSUE TITLE
Bug report: Can't call numpy() on Tensor that requires grad. Use tensor.detach().numpy() instead.

### ISSUE THREAD

**Issue by TihayaKousaka** [TihayaKousaka/ML_Method#4]
# bug
所有调用numpy()的都加了.detach()，测试程序依然报错
